File: Daytwo/python/task_seven.py
def sum_element_of_odd_index(arrayPassedIn):
	evenIndex = [];
	sum = 0;
	int counter = 0;
	for number in range 10:
		if(number%2==1):
			evenIndex[counter] = arrayPassedIn[number]
			sum+=evenIndex[counter]
			counter+=1
	return sum

File: Daytwo/java/TaskSeven.java
public class TaskSeven{
	public static int sumOfElementOfOddIndex(int[] arrayPassedIn){
		int sum = 0;
		for(int count = 0; count<10;count++){
			if(count%2==1){
				sum += arrayPassedIn[count];
			}
		}
	return sum;
	}
}